code/modules/projectiles/guns/projectile/bow.dm

```diff
@@ -96,3 +107,85 @@
 	..()
 	for(var/i in 1 to storage_slots)
 		new /obj/item/ammo_casing/caseless/arrow(src)
+
+/obj/item/weapon/storage/backpack/quiver/proc/arrow_count()
+	var/count = 0
+	for(var/obj/item/ammo_casing/caseless/arrow/A in src)
+		count++
```

Comment:
Uh, for quivers as a whole? Because the normal one has no need to track arrow amounts